conf/app.src

```diff
@@ -1,5 +1,5 @@
-SOURCE_URL=https://github.com/getgrav/grav/releases/download/1.5.3/grav-admin-v1.5.3.zip
-SOURCE_SUM=055F6F248AA74C68436B6D639D9340D333136044E5946FC6D62271AD2B3B319B
+SOURCE_URL=https://github.com/getgrav/grav/releases/download/1.5.8/grav-admin-v1.5.8.zip
+SOURCE_SUM=3b0210bf8639c1d75b046e4a9b364b216b41c449c8b5ac74fd71c098d25cb9bb
```

Comment:
The sums don't matct? It's different to the one in app-upgrade.src?

Not sure if that is meant to be the case ;)